Dockerfile

```diff
@@ -1,5 +1,9 @@
 # Build the manager binary
-FROM golang:1.20 as builder
+FROM registry.access.redhat.com/ubi8-minimal:latest as builder
+ARG GO_PLATFORM=amd64
+env PATH=$PATH:/usr/local/go/bin
```

Comment:
Should this be `ENV` instead of `env`?

Reply:
Yes. It seems to work with 'env' but that isnt' documented, so I've updated it.